### PR TITLE
adding cmi5.xml file to required folder

### PR DESCRIPTION
**Pull request by bell87**
@@ strings get replaced with json property values from course.json within grunt-replace task

### CODE REVIEW

File: required/cmi5.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<courseStructure xmlns="http://www.adlnet.gov/cmi5/CourseStructure.xsd">
+    <course id="">
+        <title>
+            <langstring lang="en-US">@@title</langstring>
+        </title>
+        <description>
+            <langstring lang="en-US">@@body</langstring>
+        </description>
+    </course>
+    <au id="">
```
Comment (ryangadams) [inline required/cmi5.xml]: is this id the ActivityId? It needs to come from the json too. Not sure where it's to be defined.

Comment (bell87) [inline required/cmi5.xml]: Yes, I think so - https://github.com/AICC/CMI-5_Spec_Current/blob/quartz/cmi5_spec.md#id

**Comment by brian-learningpool** [comment]
+1 and we'll come back to the `id` value.

**Comment by davidpesce** [comment]
Hold off on the ID value. We are working out something that may change how it's created.

**Comment by ryangadams** [comment]
In that case I +1 too…

**Comment by ryangadams** [comment]
@davidpesce would you mind elaborating on your id value comment? is that a CMI5 thing where you may not bundle the ActivityId in the xml?

**Comment by davidpesce** [comment]
It will definitely be there, but how it's generated may change. There is talk of attempting to make it globally unique (ie hash of course structure). There's also discussion around whether or not there will be an originalID, which would indicate if content was modified after the content developer published it. 

I'm guessing we will have it solidified this week or next.

**Pull request merged**